app/code/Magento/Eav/Model/Entity/Increment/NumericValue.php

```diff
@@ -28,7 +28,7 @@ public function getNextId()
     {
         $last = $this->getLastId();
 
-        if (strpos($last, (string) $this->getPrefix()) === 0) {
+        if ($last && strpos($last, (string) $this->getPrefix()) === 0) {
```

Comment:
shouldn't we check if it's a non-empty string, like that?
```suggestion
        if (is_string($last) && '' !== $last && strpos($last, (string)$this->getPrefix()) === 0) {
```
```suggestion
        if ($last && strpos($last, (string) $this->getPrefix()) === 0) {
```

Reply:
Hello @ihor-sviziev 
Proposed changes are well for performance.
Thank you for it.
@xmav What do you think about proposed changes.
Thank you in advance for answer.